packages/connect-node-test/src/handler.spec.ts

```diff
@@ -54,7 +53,9 @@ describe("createHandler()", () => {
       TestService.methods.emptyCall,
       (req) => req,
       {
-        protocols: [createConnectProtocol({})],
+        connect: true,
+        grpcWeb: false,
+        grpc: false,
```

Comment:
This PR changes the public API for creating handlers. The functions `createXProtocol()` are no longer available. 
All protocols are still enabled by default. You can disable individual protocols with a boolean property.